requirements/adg/base.txt

```diff
@@ -1 +1,3 @@
 mandrill==1.0.57
+mailchimp3==3.0.14
+pytest-mock==3.3.1
```

Comment:
Let's please make sure that these new packages are not downgrading or upgrading any existing package.

Reply:
There is no change in any package, due to packages added by me.

**But** in another PR we have added `mandrill==1.0.57` which is downgrading `docopt==0.6.2` to `docopt==0.4.0`. This may break something in future. My suggestion is to use latest version `mandrill==1.0.59`

Reply:
@muak50 can we please upgrade mandrill along with the unit tests we are planning to write?

Reply:
Sure, will do and test if everything's working fine.